homeassistant/components/permobil/config_flow.py

```diff
@@ -161,17 +161,12 @@ async def async_step_email_code(
         return self.async_create_entry(title=self.data[CONF_EMAIL], data=self.data)
 
     async def async_step_reauth(
-        self, user_input: Mapping[str, Any]
+        self, entry_data: Mapping[str, Any]
     ) -> ConfigFlowResult:
         """Perform reauth upon an API authentication error."""
-        reauth_entry = self.hass.config_entries.async_get_entry(
-            self.context["entry_id"]
-        )
-        assert reauth_entry
-
```

Comment:
The entry data is already available in `entry_data` for all reauth flows.
Thus, there is no need to do a lookup on the entry just to get the entry data.